.github/workflows/release.yml

```diff
@@ -156,3 +156,19 @@ jobs:
         asset_path: pvsadm-binaries.tar.gz
         asset_name: pvsadm-binaries.tar.gz
         asset_content_type: application/tar+gzip
+
+    - name: Update Homebrew formula
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
+        GH_TOKEN: ${{ secrets.GH_TOKEN}}
+      run: |
+        cd ci/
+        chmod +x brew_formula_updater.sh
```

Comment:
can you change the permission while pushing this file itself?

Reply:
Done